communication/moderation.md

```diff
@@ -0,0 +1,48 @@
+# Moderation on Kubernetes Communications Channels
+
+This page describes the rules and best practices for people chosen to moderate Kubernetes communications channels, such as Slack and the mailing lists.
+
```

Comment:
should we specifically call out every channel that we moderate that would fall under the coc umbrella so that it's explicit? ie - zoom meetings (sig chairs are responsible for moderating), discourse (even though its just testing), etc.?

Reply:
I wanted to keep it generic enough so that we didn't need to edit this document everytime we have a presence somewhere. I'll make it generic so that it applies to everything we do like the CoC does.